lithium/test_lithium.py

```diff
@@ -510,6 +511,29 @@ def test_char_dd(self):
         self.assertEqual(t.parts, [b"d", b"a", b"t", b"a", b"\n", b"2"])
         self.assertEqual(t.after, b"\nDDEND\npost\n")
 
+    def test_jsstr(self):
```

Comment:
Because we currently have the "missing-docstring" pylint suppression file-wide, so this was not flagged, but I think we should incrementally add docstrings for new code going forward.